lockfile_generator/src/lib.rs

```diff
@@ -184,8 +185,11 @@ impl Display for Error {
             Self::UnsupportedCommandVersion(command, expected, received) => {
                 write!(f, "unsupported {command:?} version {received:?}, expected {expected:?}")
             },
-            Self::NoLockfileGenerated => write!(f, "no lockfile was generated"),
             Self::Anyhow(err) => write!(f, "{err}"),
+            Self::UnexpectedOutput(cmd, output) => {
+                write!(f, "unexpected output for {cmd:?}: {output}")
+            },
+            Self::NoLockfileGenerated => write!(f, "no lockfile was generated"),
```

Comment:
If the goal was to keep these in the same order as above, you'll need to flip `Anyhow` and `UnexpectedOutput`

```suggestion
            Self::UnexpectedOutput(cmd, output) => {
                write!(f, "unexpected output for {cmd:?}: {output}")
            },
            Self::Anyhow(err) => write!(f, "{err}"),
            Self::NoLockfileGenerated => write!(f, "no lockfile was generated"),
```